js/components/volumePanel.js

```diff
@@ -0,0 +1,79 @@
+const React = require('react');
+const PropTypes = require('prop-types');
+const ControlBar = require('./controlBar');
+
+/**
+ * Volume menu. This component is used for both the Popover and
+ * Screen modes of the menu.
+ */
+class VolumePanel extends React.Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+
+    return (
+      <div className="oo-absolute-centered">
```

Comment:
Can we add a class name that identifies this component, such as `oo-volume-panel`. This will make it easier for customers to customize the styles of this component.

Reply:
Missed this, will add.